pkg/dnsbl/dnsbl.go

```diff
@@ -75,7 +80,7 @@ func (c *Checker) query(ipAddress, bl string, addresses chan<- int) {
 		log.Printf("%v present in %v(%v)", reversedIPAddress, bl, result)
 	}
 	addresses <- len(result)
```

Comment:
The minimum impact might be in changing this value to something like `min { 1, len(result) }`.